content/blog/2016-07-23-pull-does-not-scale-or-does-it.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Pull doesn't scale - or does it?
+created_at: 2016-07-23
+kind: article
+author_name: Julius Volz
+---
+
+Let's talk about one particularly hard-to-kill myth again. Whenever there is a
```

Comment:
Yes, that's why I compared it to the word "hacker". I'm pretty sure the most common usage is the one for a criminal. It's your blog post, of course. I just tried to convey what a domain expert of myths feels if you use the word in the common way. Your decision if you like to make the experts feel that way or if the common usage is good enough for your standards.